spatialos-sdk/src/worker/component.rs

```diff
@@ -94,6 +94,11 @@ impl UpdateParameters {
         Default::default()
     }
 
+    /// Enables loopback short-circuiting for the component update message.
+    ///
+    /// [See the docs for more information.][short-circuit]
+    ///
+    /// [short-circuit]: https://docs.improbable.io/reference/13.5/shared/design/commands#properties
```

Comment:
This link isn't quite correct. I don't think we have a link for this one yet since the C API doesn't have full docs and this isn't exposed in the other SDKs yet.

Maybe just put a TODO here to put a link when its available?

Reply:
Done!